resource/in

```diff
@@ -7,6 +7,7 @@ from moj_analytics.auth0_client import (
     Auth0,
     Client,
     ManagementAPI,
+    log
```

Comment:
It's fine but it makes me wonder if it's a bit odd that the logger is in `moj_analytics.auth0_client` module (if you know what I mean).

It makes me wonder if it could go in its own little module, but I don't know if you think it may be overkill.

Reply:
Yeah... I did it this way so I could test things. Will tidy up later. ;-)